storage/volume/driver/driver_interface.go

```diff
@@ -26,6 +26,9 @@ type Driver interface {
 type Opt interface {
 	// Options return module customize volume options.
 	Options() map[string]types.Option
+
+	// Config is used to pass the daemon volume configure into driver.
+	Config(Context, map[string]interface{}) error
```

Comment:
I think the Driver interface should implement Config method, not the Opt interface. WDYT? @rudyfly

Reply:
I have refacted it, PTAL 😄